Client/Frontend/Browser/BrowserViewController/BrowserViewController+KeyCommands.swift

```diff
@@ -133,7 +133,9 @@ extension BrowserViewController {
     }
     
     @objc private func showDownloadsKeyCommand() {
-        navigationHelper.openDownloads()
+        navigationHelper.openDownloads() { [weak self] success in
+            self?.displayOpenDownloadsError()
+        }
```

Comment:
Not checking `success` here

Reply:
Fix at https://github.com/brave/brave-ios/pull/4769/commits/eea962dee941396138b044b0054c80076e83d780